src/types.ts

```diff
@@ -3299,7 +3309,8 @@ export enum HostEvent {
      * This event is not supported in visualization embed and search embed.
      * @param - object - To trigger the action for a specific visualization
      * in Liveboard embed, pass in `vizId` as a key.
-     * **Required** in Spotter embed.
+     * @param - `vizId` refers to the answer id in spotter Embed, it is required in spotter Embed.
```

Comment:
`vizId` refers to the Answer ID in Spotter Embed and is required in Spotter Embed.